Examples/Tests/projection_divb_cleaner/inputs_rz

```diff
@@ -0,0 +1,51 @@
+warpx.serialize_initial_conditions = 0
```

Comment:
I think for this and the `PICMI_inputs_3d` test in the same folder it would be better to modify the original tests instead of creating new ones. I say this because we know the existing tests are strictly less accurate than these new ones - not having divergence free B-fields are purely bad and therefore I don't see a good reason to keep both sets of tests.

Reply:
So to be clear you want to merge the external field loader from file tests and have a separate one for the python loader?

Reply:
Yes, that sounds good to me.

Reply:
@dpgrote When we talked earlier you thought that we should leave the default to off for the clean_divb_external flag. I think we should have a quick conversation between you, me, and @roelof-groenewald about it. I set it up to default to off. If we want to default it to on, then some of the other external field unit tests likely will need to have updated benchmarks. What behavior do we desire here?

Reply:
@roelof-groenewald Since the existing tests are for collocated grids, the cleaner with its default of being on will log a warning. The new CI tests added are on a Yee mesh, which will use the cleaner. I think covering all the options is best. I need to probably do some additional testing to make sure it is working properly with the collocated grids, but that could be another PR.